changelog.md

```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
 and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html)
+## [Unreleased]
+
+### Added
```

Comment:
Missing guides migration update, to explain that now with have to set itemtype  on Array type with some examples. Please read other migration guide to know how to write it.

Reply:
done